src/main/java/nextstep/subway/auth/application/AuthService.java

```diff
@@ -20,7 +20,7 @@ public AuthService(MemberRepository memberRepository, JwtTokenProvider jwtTokenP
     }
 
     public TokenResponse login(TokenRequest request) {
-        Member member = memberRepository.findByEmail(request.getEmail()).orElseThrow(AuthorizationException::new);
+        Member member = memberRepository.findByEmail(request.getEmail()).orElseThrow(() -> new AuthorizationException("유효하지 않는 계정입니다."));
```

Comment:
```suggestion
        Member member = memberRepository
        .findByEmail(request.getEmail())
        .orElseThrow(() -> new AuthorizationException("유효하지 않는 계정입니다."));
```
이렇게 개행을 해주면 더 좋을 것 같습니다!